index.js

```diff
@@ -1,29 +1,30 @@
-
-const core   = require('@actions/core');
+const core = require('@actions/core');
 //const github = require('@actions/github');
 const semver = require('semver');
 
 try {
-	const since_perl = semver.coerce (core.getInput('since-perl'));
-	const with_devel = core.getInput('with-devel') == "true";
+    const since_perl = semver.coerce(core.getInput('since-perl'));
+    const with_devel = core.getInput('with-devel') == "true";
 
-	let available = [
-		"5.8",
-		"5.10", "5.12", "5.14", "5.16", "5.18",
-		"5.20", "5.22", "5.24", "5.26", "5.28",
-		"5.30", "5.32", "5.34", "5.36", "5.38",
-		"devel",
-	];
+    let available = [
+        "5.8",
+        "5.10", "5.12", "5.14", "5.16", "5.18",
+        "5.20", "5.22", "5.24", "5.26", "5.28",
+        "5.30", "5.32", "5.34", "5.36", "5.38",
+        "devel",
+    ];
 
-	let filtered = available.filter (
-		(item) => {
-			if (item == "devel") { return with_devel; }
-			return semver.gte(semver.coerce(item), semver.coerce(since_perl));
-		}
-	);
+    let filtered = available.filter(
+        (item) => {
+            if (item == "devel") {
+                return with_devel;
+            }
+            return semver.gte(semver.coerce(item), semver.coerce(since_perl));
+        }
+    );
 
-	console.log('perl-versions', JSON.stringify(filtered));
-	core.setOutput('perl-versions', JSON.stringify(filtered));
+    console.log('perl-versions', JSON.stringify(filtered));
+    core.setOutput('perl-versions', JSON.stringify(filtered));
 } catch (error) {
-	core.setFailed(error.message);
-}
+    core.setFailed(error.message);
+}
```

Comment:
please also configure tooling to enforce trailing newline

Reply:
same this is coming from `pretty` I'm sure this is configurable, not sure how, feel free to adjust